blockdag/dag.go

```diff
@@ -718,81 +717,86 @@ func addTxToMultiset(ms *secp256k1.MultiSet, tx *wire.MsgTx, pastUTXO UTXOSet, b
 func (dag *BlockDAG) saveChangesFromBlock(block *util.Block, virtualUTXODiff *UTXODiff,
 	txsAcceptanceData MultiBlockTxsAcceptanceData, feeData compactFeeData) error {
 
-	databaseTx, err := dbaccess.NewTx()
+	// Atomically insert info into the database.
```

Comment:
I think this comment is redundant. You don't need to remind developers why transactions are needed